CONDUCT.md

```diff
@@ -24,6 +24,13 @@ Examples of unacceptable behavior by participants include:
 
 ## Our Responsibilities
 
+Our team members are committed to ensuring an inclusive environment and addressing any behaviour that does not align with the Code of Conduct. Our team members currently include:
+
+* LG Nguyen
```

Comment:
hi Steven, could you change my name here to Linh Giang Nguyen (that's how it's officially recorded hehe)? thanks!

Reply:
Thanks for pointing out.  Changed and committed.